[Durable Objects] websocket hibernation (#7872)
cloudflare/cloudflare-docs: content/workers/runtime-apis/websockets.md

## Background

> WebSockets allow you to communicate in real time with your Cloudflare Workers serverless functions. For Durable Objects, prefer using the [WebSockets Hibernation API](/workers/runtime-apis/durable-objects/#websockets-hibernation-api).

Reviewer: Could we also add a note here or on this page that clarifies that the `WebSocket` object, which is otherwise documented here, has additional methods on it when used with Durable Objects? Just to cover the case where someone ends up looking here for these, when they really should be looking at `/runtime-apis/durable-objects/`

> WebSockets allow you to communicate in real time with your Cloudflare Workers serverless functions. For Durable Objects, prefer using the [WebSockets Hibernation API](/workers/runtime-apis/durable-objects/#websockets-hibernation-api) which adds Cloudflare-specific extensions to the standard `WebSocket` object.

Reviewer: ```suggestion
WebSockets allow you to communicate in real time with your Cloudflare Workers serverless functions.

When using WebSockets with Durable Objects, we recommend using the [WebSockets Hibernation API](/workers/runtime-apis/durable-objects/#websockets-hibernation-api) which adds additional extensions to the standard `WebSocket` object. 
```
